cmd/scollector/main.go

```diff
@@ -134,6 +134,9 @@ func main() {
 	for _, r := range conf.Riak {
 		check(collectors.Riak(r.URL))
 	}
+	for _, cfg := range conf.Database {
```

Comment:
This is the old way of initializing things. For new things we have a different way to do it. See https://github.com/bosun-monitor/bosun/blob/master/cmd/scollector/collectors/haproxy_unix.go#L16 for an example. This lets us add new things without touching main.